src/external/hello.js

```diff
@@ -0,0 +1,3 @@
+global.hello = function() {
```

Comment:
Let's remove this file – the other one can be used as an example.

Reply:
Sure